### PR TITLE
Adds content for Advance.

### PR DESCRIPTION
**Pull request by chuckharmston**
home

detail

example-screenshot

### CODE REVIEW

**Comment by chuckharmston** [comment]
(@flodolo this is so WIP that there's no need for review; I'll ping you when we're ready to merge.)

**Comment by pdehaan** [comment]
@chuckharmston Interesting! Can you ping me the repo URL for Laserlike? I couldn't find it in GitHub org (unless it's private or something sneaky sneaky).

File: content-src/experiments/laserlike.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+id: 18
```
Comment (pdehaan) [inline content-src/experiments/laserlike.yaml]: Do we need some `dev: true` style footgun protection so this doesn’t accidentally land on prod?

Comment (chuckharmston) [reply content-src/experiments/laserlike.yaml]: Good call! ✅

**Pull request merged**